src/generic/listctrl.cpp

```diff
@@ -3403,10 +3416,9 @@ wxListMainWindow::ComputeMinHeaderWidth(const wxListHeaderData* column) const
     const int image = column->GetImage();
     if ( image != -1 )
     {
-        if ( m_small_image_list )
+        if ( m_small_images )
         {
-            int ix = 0, iy = 0;
-            m_small_image_list->GetSize(image, ix, iy);
+            const int ix = m_small_images->GetImageLogicalSize(this, image).GetWidth();
```

Comment:
We could minimize the code churn here as well:
```suggestion
            int ix = 0, iy = 0;
            m_small_images->GetImageLogicalSize(this, image, ix, iy);
```